src/CommunityToolkit.Maui.Core/Views/RoundedStackView.macios.cs

```diff
@@ -57,21 +58,21 @@ public override void Draw(CGRect rect)
 		Layer.MasksToBounds = true;
 	}
 
-	static CGPath? GetRoundedPath(CGRect rect, nfloat left, nfloat top, nfloat right, nfloat bottom)
+	static CGPath? GetRoundedPath(CGRect rect, NFloat left, NFloat top, NFloat right, NFloat bottom)
 	{
 		var path = new UIBezierPath();
 		path.MoveTo(new CGPoint(rect.Width - right, rect.Y));
 
-		path.AddArc(new CGPoint((float)rect.X + rect.Width - right, (float)rect.Y + right), (nfloat)right, (float)(Math.PI * 1.5), (float)Math.PI * 2, true);
+		path.AddArc(new CGPoint((float)rect.X + rect.Width - right, (float)rect.Y + right), (NFloat)right, (float)(Math.PI * 1.5), (float)Math.PI * 2, true);
 		path.AddLineTo(new CGPoint(rect.Width, rect.Height - bottom));
 
-		path.AddArc(new CGPoint((float)rect.X + rect.Width - bottom, (float)rect.Y + rect.Height - bottom), (nfloat)bottom, 0, (float)(Math.PI * .5), true);
+		path.AddArc(new CGPoint((float)rect.X + rect.Width - bottom, (float)rect.Y + rect.Height - bottom), (NFloat)bottom, 0, (float)(Math.PI * .5), true);
 		path.AddLineTo(new CGPoint(left, rect.Height));
 
-		path.AddArc(new CGPoint((float)rect.X + left, (float)rect.Y + rect.Height - left), (nfloat)left, (float)(Math.PI * .5), (float)Math.PI, true);
+		path.AddArc(new CGPoint((float)rect.X + left, (float)rect.Y + rect.Height - left), (NFloat)left, (float)(Math.PI * .5), (float)Math.PI, true);
 		path.AddLineTo(new CGPoint(rect.X, top));
 
-		path.AddArc(new CGPoint((float)rect.X + top, (float)rect.Y + top), (nfloat)top, (float)Math.PI, (float)(Math.PI * 1.5), true);
+		path.AddArc(new CGPoint((float)rect.X + top, (float)rect.Y + top), (NFloat)top, (float)Math.PI, (float)(Math.PI * 1.5), true);
```

Comment:
Let's update all of the casts to use `NFloat` instead of `float`.

That'll allow us to use the overloaded method, `AddArc(GCFloat, NFloat, NFloat, NFloat, bool)`:

<img width="835" alt="image" src="https://user-images.githubusercontent.com/13558917/158698871-5b5216f0-5d2f-4da0-8e50-b0fe228db161.png">